src/cli.ts

```diff
@@ -2,33 +2,67 @@
 
 import fs from "node:fs";
 import path from "node:path";
+import { parseArgs } from 'node:util'
+import { createRequire } from 'node:module'
+import * as process from 'process'
```

Comment:
Can we make it this for consistency?
```suggestion
import process from 'node:process'
```

Reply:
My mistake, process is a global var